src/main/java/edu/greenblitz/robotName/subsystems/shooter/Funnel/NeoFunnel/NeoFunnelConstants.java

```diff
@@ -0,0 +1,24 @@
+package edu.greenblitz.robotName.subsystems.shooter.Funnel.NeoFunnel;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkMaxLimitSwitch;
+import edu.greenblitz.robotName.RobotConstants;
+import edu.greenblitz.robotName.utils.motors.GBSparkMax;
+
+public class NeoFunnelConstants {
+    public static final int FUNNEL_ID = 0;
+
+    public static final boolean IS_SENSOR_EXIST = true;
+
+    public static final SparkMaxLimitSwitch.Type SWITCH_TYPE = SparkMaxLimitSwitch.Type.kNormallyClosed;
+
+    public static double DEBOUNCE_TIME_FOR_LIMIT_SWITCH = 0.5;
```

Comment:
Thats half a second. I think that the others are at 0.2 or less. Check in Charrrged-Up